tests/tests.rs

```diff
@@ -246,7 +247,17 @@ async fn test_work_dir_lock() {
 
             let storage = res_one.unwrap();
             let result = waitpid(child, None);
+            #[cfg(not(target_os = "macos"))]
             assert_eq!(Ok(WaitStatus::Exited(child, 0)), result);
```

Comment:
This test is incorrect. `fork` splits test engine as well, so the forked process can continue running other tests. Probably, that is the reason why the exit code is `0` (on panic it should not be 0)

Reply:
Please, update to use `rusty-fork` crate (https://docs.rs/rusty-fork/0.3.0/rusty_fork/fn.fork.html)

Reply:
Actually, lets do this in a separate request. I added a new issue: https://github.com/qoollo/pearl/issues/254 and assigned it to you